src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -245,10 +264,10 @@ AND NOT EXISTS (select prs from p.results prs where prs.assessor.id = p.student.
     @Query("""
             select distinct p from StudentParticipation p left join fetch p.submissions s left join fetch s.results r
             where p.exercise.id = :#{#exerciseId}
+            and (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
            and p.testRun = FALSE
```

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c